appengine-vmruntime/vmruntime/wsgi.py

```diff
@@ -50,6 +51,22 @@
 # Ensure API requests include a valid ticket by default.
 vmstub.Register(vmstub.VMStub(env_config.default_ticket))
 
+# Enable Python Cloud Debugger if one is present. The debugger agent
+# communicates with the Cloud Debugger backend.
+#
+# The debugger has no effect on the application until snapshot is
+# requested.
+#
+# For details see: https://github.com/GoogleCloudPlatform/cloud-debug-python
+try:
+  import googleclouddebugger
+  googleclouddebugger.AttachDebugger()
+except ImportError:
+  pass
+except Exception as e:
+  logging.warn('Exception while initializing Cloud Debugger: %s',
```

Comment:
That would log it as an error, right? I don't want to imply that a failure to start the debugger is an error in the program.